x-pack/plugins/reporting/server/routes/lib/job_response_handler.ts

```diff
@@ -85,8 +87,11 @@ export async function deleteJobResponseHandler(
     });
   }
 
+  const docIndex = doc.index;
+  const stream = await getContentStream(reporting, { id: docId, index: docIndex });
+
   try {
-    const docIndex = doc.index;
+    await promisify(stream.end.bind(stream))();
```

Comment:
a comment explaining this code would be helpful :) the behaviour of removing chunks seems a bit implicit.